CExcercises/Ex1-nIndirections.c

```diff
@@ -16,22 +16,23 @@ El resultado será 3.*/
 int nIndirections(void* initial, unsigned int indirections);
 
 int main (int argc, char **argv)
-{
+{	
+	int indirections = 2; // To not use magical values
 	int a = 3;
 	int *a1 = &a;
 	int **a2 = &a1;
-	int result = nIndirections((void*)a2, 2);
+	int result = nIndirections((void*)a2, indirections);
 	printf("The result of 2 indirections is %i \n", result);
 	return 0;
```

Comment:
Instead of returning 0, check this http://www.cplusplus.com/reference/cstdlib/EXIT_SUCCESS/